code/modules/urist/gamemodes/infestation/weapons&armour.dm

```diff
@@ -167,11 +167,12 @@
 	item_state = "ANFOR-rifle"
 	w_class = 4
 	force = 10
-	caliber = CALIBER_RIFLE
```

Comment:
This change requires the same be made to its magazine here:
https://github.com/UristMcStation/UristMcStation/blob/be69851be262cbf2f7d7e77d85c134e80255f3ec/code/modules/urist/gamemodes/infestation/weapons%26armour.dm#L199